### ISSUE TITLE
human reads

### ISSUE THREAD

**Issue by LisaHollstein** [MHH-RCUG/nf_wochenende#57]
Right now the part that removes the human reads from the tables is missing, therefore those reads are present in the haybaler tables.

**Comment by colindaven** [comment]
You mean this script, correct ?    
`bash runbatch_metagen_window_filter.sh`

I wonder where the old Wochenende_postprocess removes these human reads (a haybaler section perhaps?),
because reporting uses `bam.txt` files (unfiltered) and haybaler uses reporting results.

**Comment by LisaHollstein** [comment]
Yeah, I meant that script. Interestingly, it seems to be run after haybaler in the Wochenende_postprocess, so I don't really know how there were no human reads in the haybaler csvs. 

So my question is: Should there be human reads in the haybaler  csvs or not? (I remember seeing csvs with them (a year ago) and some without (more recent))

**Comment by colindaven** [comment]
I think Sophia excluded human reads in the python or bash scripts, maybe because she needed the numbers for normalization first, but not sure. Normalization should be done by the reporting steps. 

You can see with this search that they are excluded with `grep -v` for the heatmaps and heat trees.

https://github.com/MHH-RCUG/haybaler/search?q=1_1_1

**Comment by colindaven** [comment]
Ilona will know best if the human info is completely excluded or not and since when.
